modules/packages/Crypto.chpl

```diff
@@ -41,101 +42,158 @@ module Crypto {
           "openssl/aes.h", "openssl/rand.h", "-lcrypto";
   //require "CryptoHandlers/rsa_complex_bypass_handler.h";
 
+  pragma "no doc"
```

Comment:
When interfacing with C libraries, there is a common pattern to consolidate the C interface into a submodule, typically named `C_<library>`, e.g. [BLAS.C_BLAS](http://chapel.cray.com/docs/master/modules/packages/BLAS/C_BLAS.html) of the [BLAS](http://chapel.cray.com/docs/master/modules/packages/BLAS.html) module.

This might be a nice way to keep these `extern` symbols documented, but also separated from the intended user-facing documentation, e.g. `Crypto.C_OpenSSL`.